backend/core/confload/confload.py

```diff
@@ -87,13 +92,31 @@ def setup_logging(self, max_debug=False):
         logging.config.dictConfig(log_config_dict)
         log.info(f"Logging setup @ {__name__}")
 
+    @property
+    def project_root(self):
+        config_file_path = Path(self.config_filename).absolute()
+        return str(config_file_path.parent)
+
+    def find_actual_tfsm_path(self):
+        potentials = [
+            'backend/plugins/extensibles/ntc-templates/index',
+            '/code/backend/plugins/extensibles/ntc-templates/index',
+            '/usr/local/lib/python3.8/site-packages/ntc_templates/templates/index'
+        ]
```

Comment:
This is kind of hacky, but we can get more sophisticated if this proves insufficient

Reply:
good enough for now